src/CustomerProfile.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+
+const CustomerProfile = (props) => {
+  const [customer, setCustomer] = useState("");
+
+  let id = props.id;
```

Comment:
Think we can use `const` here, always best to use const when we know our variable is not going to reassigned.